getUser/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace getUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            User user = new User();
+
+            Console.WriteLine($"Hello stranger, I have a few  questions for you: ");
+
+            user.FirstName = user.AnswerQuestion("What is your first name?");
+            user.LastName = user.AnswerQuestion("What is your last name?");
+            user.FaveAnimal = user.AnswerQuestion("What is your favorite animal?");
+            user.FaveSport = user.AnswerQuestion("What is your favorite sport?");
+            user.FaveNumber = user.AnswerQuestionWithInteger("What is your favorite number?");
+
+            Console.WriteLine($"Hello {user.FirstName} {user.LastName}.");
+            Console.WriteLine($"Your favorite animal is a {user.FaveAnimal}.");
+            Console.WriteLine($"Your favorite sport is {user.FaveSport}.");
+            Console.Write($"And your favorite number is");
+
+            if(user.FaveNumber < 5 )
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine($" {user.FaveNumber}");
```

Comment:
Perhaps the space could be at the end of the string on line 22, then just have `Console.WriteLine(user.FaveNumber);`?